dbfit-java/core/src/main/java/dbfit/util/sql/PreparedStatements.java

```diff
@@ -8,11 +8,9 @@ private static String buildParamList(int numberOfParameters) {
         return join(repeat("?", numberOfParameters), ", ");
     }
 
-    public static String buildStoredProcedureCall(String procName, int numberOfPararameters) {
-        return "{ call " + procName + "(" + buildParamList(numberOfPararameters) + ")}";
-    }
-
-    public static String buildFunctionCall(String procName, int numberOfParameters) {
-        return "{ ? = call " + procName + "(" + buildParamList(numberOfParameters - 1) + ")}";
+    public static String buildStoredRoutineCallCmdText(
```

Comment:
Either `buildStoredRoutineCallCommand` or `buildStoredRoutineCallText` might do. I think `buildStoredRoutineCallStatementText` is the most expressive.